### DISCUSSION TITLE
cuando creas una nueva wallet tenes que esperar que sincronice la wallet es diferente la sincronizacion del nodo blockchain

### DISCUSSION THREAD

**Discussion by nenedelnorte** [Chia-Network/chia-blockchain#5101] [Q&A]
de https://www.chiastatus.com/addresses/xch177jdel6rgvzdh3hwzlusmkh0vtmcxl8ns5z3kx9y6wfn4u5wzmfqkaexdr
A https://www.chiastatus.com/addresses/xch1l88ftq403uv2h0rszn3gdgnc8dlpyc8ps3pyry98fxcvuadvj5kq4p86d7

monedero no actualiza, tengo las 24 palabras saldo sin movimiento.
![image](https://user-images.githubusercontent.com/84234325/118363771-6a554280-b56c-11eb-9ab2-7c191ff8f2ce.png)

**Comment by nenedelnorte** [comment]
![image](https://user-images.githubusercontent.com/84234325/118364160-a9d05e80-b56d-11eb-8550-0c6f2d97cb64.png)

**Reply by stravoskyus** [reply]
a mi me a pasado exactamente lo mismo y ya hace 14h de la transferencia entre mis monederos de chia coin

**Reply by stravoskyus** [reply]
el monedero esta conectado y no e recibido nada aun, en chia explores sale la tranferencia con este mensaje: ⚠️ Chia Explorer puede mostrar un resultado neto diferente a su billetera Chia y ambos son correctos . Cuando envía monedas, el cambio puede ir a una dirección diferente. La billetera conoce todas sus direcciones y Chia Explorer no. Actualmente está buscando en una dirección específica.

**Reply by stravoskyus** [reply]
alguien me puede ayudar? con lo que me a constado conseguir 2 XCH .....

**Reply by nenedelnorte** [reply]
espera que tu wallet descargue el nodo completo
![image](https://user-images.githubusercontent.com/84234325/118376984-9b039f00-b5a1-11eb-8949-6ab93bac1047.png)

**Reply by nenedelnorte** [reply]
https://www.chiastatus.com/

**Comment by nenedelnorte** [comment]
https://www.chiastatus.com/addresses/xch1uzg8zynf2p2n5q5d6ccepng3tleefg5z5ezht23un8ryueyrw6ysaa229s

**Comment by RRASHH** [comment]
Seguramente que sea porque el nuevo monedero se tiene que sincronizar. Ahora mismo está tardando casi un día entero. Fíjate en el estado que te pone en la pestaña del nuevo monedero que has creado.

**Reply by nenedelnorte** [reply]
apague todas mi maquina deje una sola sincronizando la nueva wallet 1.1.5

**Reply by nenedelnorte** [reply]
foto ver pestaña estado?

**Comment by nenedelnorte** [comment]
ya entendi. el nodo es una sincronizacion, wallet tiene otra sincronizacion. granja otra sincronizacion.- gracias RRASHH GRgRGRgrggrgr

**Comment by stravoskyus** [comment]
me a pasado lo mismo hace 14 horas

**Reply by nenedelnorte** [reply]
![Uploading image.png…]()

**Comment by stravoskyus** [comment]
y el monedero esta sincronizado, es una transferencia entre mis 2 monederos de chiacoin blokchain

**Reply by nenedelnorte** [reply]
![image](https://user-images.githubusercontent.com/84234325/118376991-a6ef6100-b5a1-11eb-986c-8953447b35dd.png)

**Comment by stravoskyus** [comment]
⚠️ Chia Explorer puede mostrar un resultado neto diferente a su billetera Chia y ambos son correctos . Cuando envía monedas, el cambio puede ir a una dirección diferente. La billetera conoce todas sus direcciones y Chia Explorer no. Actualmente está buscando en una dirección específica.

**Reply by nenedelnorte** [reply]
![image](https://user-images.githubusercontent.com/84234325/118377001-b373b980-b5a1-11eb-927a-0c8f1ddeda70.png)

**Comment by stravoskyus** [comment]
este mensaje sale con la direccion de monedero . sale la cantidad, 2 XCH,enviado ,
 pero no llega nada..............

**Reply by nenedelnorte** [reply]
![image](https://user-images.githubusercontent.com/84234325/118376998-b1115f80-b5a1-11eb-9b92-67522c37e45b.png)

**Reply by nenedelnorte** [reply]
cada wallet tiene que sincronizar desde 0 conviene crear wallet asi se descarga desde ahora en 0xch para tenerla lista. o usar billletera calientes.

**Reply by stravoskyus** [reply]
e vuelto a reiniciar.ahora esta a la altura de 222.000

**Reply by stravoskyus** [reply]
el bloque de transferencia esta en el 283.000 creo recordar. como sincronizo desde 0?

**Reply by nenedelnorte** [reply]
son dos cosas

**Reply by nenedelnorte** [reply]
![image](https://user-images.githubusercontent.com/84234325/118379788-a4960280-b5b3-11eb-9b28-ee29af23ad40.png)

**Reply by nenedelnorte** [reply]
Estado
estado:
no sincronizado
altura:
285848 wallet

**Reply by nenedelnorte** [reply]
recuerda q pagan por tener nodo aparte farming + nodo

**Comment by nenedelnorte** [comment]
![image](https://user-images.githubusercontent.com/84234325/118379824-efb01580-b5b3-11eb-9c9a-55f453bef53e.png)

**Reply by nenedelnorte** [reply]
tiene q estar al ultimo bloque
en 270 aprox veo mi balance y luego esperar q este sincronizado

**Reply by stravoskyus** [reply]
Estado
estado:
sincronizando
altura:
224600
conexiones:
0

**Reply by nenedelnorte** [reply]
falta deja q termine y sincronize

**Reply by stravoskyus** [reply]
no entiendo mucho perdona

**Reply by nenedelnorte** [reply]
https://www.chiastatus.com verifica q sea tu wallet tiene q estar tu balance

**Reply by stravoskyus** [reply]
16 hours ago | Received 2 XCH | Transaction | 283.482 |

**Reply by nenedelnorte** [reply]
son dos nodos uno para los parcela 
y el otro para la billetera los dos tienen q estar sincronizados

**Reply by nenedelnorte** [reply]
tenes q esperar ese bloque | 283.482 
sincronizando
altura:
224600 y te falta llegar

**Reply by nenedelnorte** [reply]
tambien aprendi y tambien me asuste. cuando usamos chia programa sin darnos cuenta pasaron los dias y sincronizamos nodo y billetera por eso era rapido. ahora esperar q la nueva billetera se sincronize

**Reply by stravoskyus** [reply]
https://www.chiastatus.com/addresses/xch1tzdarenxurq83vlslm9d2e3ck7n6nvg6025fkch92509htsvly0qeu7305

**Reply by stravoskyus** [reply]
esto me sale en la direccion de mi billetera

**Reply by stravoskyus** [reply]
muchas gracias por la info ,  me siento perdido, estafado , o e hecho algo mal....

**Reply by nenedelnorte** [reply]
si tenes las 24 palabras es tuya
espera tranquilo

**Reply by stravoskyus** [reply]
las 24 palabras de las 2 cuentas tengo si

**Reply by nenedelnorte** [reply]
ahora somos parte de la red. recuerda q pagan por programa abierto y sincronizado  y por parcela es aparte. hastaq no esten los dos sincronizados. no estas conectado a la red.

**Reply by nenedelnorte** [reply]
entra en una tenes completo billetera y el otro esta cargando donde tiene q llegarte

**Reply by stravoskyus** [reply]
llegue a pensar que no se podian hacer transferencia entre 2 cuentas en la misma blockchain

**Reply by nenedelnorte** [reply]
![image](https://user-images.githubusercontent.com/84234325/118380111-d3ad7380-b5b5-11eb-8219-4b1b737c9cff.png)
mira la primera billetera esta completa y la nueva esta poco bloqueada

**Comment by stravoskyus** [comment]
de donde eres compañero, te debo una

**Reply by stravoskyus** [reply]
si me salgo de la billetera a la solapa llaves no se desconecta?

**Reply by nenedelnorte** [reply]
en la q espero q sincronize
![image](https://user-images.githubusercontent.com/84234325/118380158-15d6b500-b5b6-11eb-9a63-c8557676f0e7.png)

**Reply by stravoskyus** [reply]
y entro entre cuentas no pasa nada?

**Reply by nenedelnorte** [reply]
todo es aprender.

**Reply by stravoskyus** [reply]
espero que sea verdad por dios mio que es mucho dinero par mi

**Reply by stravoskyus** [reply]
soy español y este año esta siendo malisimo

**Reply by stravoskyus** [reply]
gracias nenedelnorte

**Reply by nenedelnorte** [reply]
tambien gane cuando tenia 1.5tb jey me paso lo mismo

**Reply by nenedelnorte** [reply]
a vos confirma lo que pienxo

**Reply by stravoskyus** [reply]
Estado
estado:
no sincronizado
altura:
229335
conexiones:
1

**Reply by stravoskyus** [reply]
tienes razon la wallet de transferencia esta a tope de altura, la que recibe es esta

**Reply by nenedelnorte** [reply]
se tiene q agregar mas billeteras para tenerlas lista para tranferencias

**Comment by stravoskyus** [comment]
pero la altura no sube

**Reply by nenedelnorte** [reply]
cierra todo reinicia y habri
 la nueva

**Reply by nenedelnorte** [reply]
fijat en crtrol alt supr q no quede nada

**Reply by stravoskyus** [reply]
abro la que recibe o la que envia

**Reply by nenedelnorte** [reply]
obvio la q recibe

**Reply by nenedelnorte** [reply]
la q envia ya esta confirmada en la bockchain ahora necesitas la billetera q se conecte completamenta

**Reply by stravoskyus** [reply]
si, entiendo

**Reply by stravoskyus** [reply]
reinicio pc o solo chia

**Reply by nenedelnorte** [reply]
ya que estas en el baile si es rapida reinicia si es lenta solo chia

**Reply by stravoskyus** [reply]
hola. esta sincronizando nodo

**Reply by stravoskyus** [reply]
deberia ir subiendo la altura de la wallet en breve ?

**Reply by stravoskyus** [reply]
cuando puede llegara tardar en horas hasta esa altura

**Reply by stravoskyus** [reply]
jajajajja soy un desastre

**Reply by nenedelnorte** [reply]
mas 24 horas

**Reply by nenedelnorte** [reply]
es la zanahoria y la mula nunca llegas

**Reply by stravoskyus** [reply]
joder que locura, entonces no toco nada y lo voy mirando, cuando llegue a la altura me deberia salir los XCH correcto?

**Reply by stravoskyus** [reply]
hace unas horas tenia ganas de meterme el palo de escoba por el culo porque soy un pelotudo

**Reply by nenedelnorte** [reply]
veraz tu chia pero recien sincronizado podes mover

**Reply by nenedelnorte** [reply]
termina de sincronizar y crea otra de 24 palabras q sincronicé

**Reply by stravoskyus** [reply]
buenas noches nene, muchas gracias por la info

**Reply by stravoskyus** [reply]
saludos desde españa

**Reply by stravoskyus** [reply]
termina de sincronizar y crea otra de 24 palabras q sincronicé, porque nene

**Reply by stravoskyus** [reply]
ya tengo 2 .

**Reply by stravoskyus** [reply]
la primera es de cobro de recompensa y la otra para enviar a otras wallets calientes

**Reply by stravoskyus** [reply]
esta bien planteado asi nene

**Reply by stravoskyus** [reply]
?
